core/ui-common/src/main/kotlin/com/wire/android/ui/theme/WireColorScheme.kt

```diff
@@ -165,13 +167,21 @@ private val LightWireColorScheme = WireColorScheme(
 
     // accents
     groupAvatarColors = listOf(
-        WireColorPalette.LightRed300, WireColorPalette.LightRed500, WireColorPalette.LightRed700,
-        WireColorPalette.LightGreen300, WireColorPalette.LightGreen500, WireColorPalette.LightGreen700,
-        WireColorPalette.LightBlue300, WireColorPalette.LightBlue500, WireColorPalette.LightBlue700,
-        WireColorPalette.LightPurple300, WireColorPalette.LightPurple500, WireColorPalette.LightPurple700,
-        WireColorPalette.LightAmber300, WireColorPalette.LightAmber500, WireColorPalette.LightAmber700,
-        WireColorPalette.LightPetrol300, WireColorPalette.LightPetrol500, WireColorPalette.LightPetrol700,
-        WireColorPalette.Gray30, WireColorPalette.Gray50, WireColorPalette.Gray70,
+        Triple(WireColorPalette.LightGreen500, WireColorPalette.LightPurple500, WireColorPalette.LightRed500),
```

Comment:
Instead of triples, it would be nicer to actually create a class so it's easier to use and we reduce the possibility of using the wrong values.

```kotlin
data class GroupAvatarColors(val left: Color, val middle: Color, val right: Color)

data class ChannelAvatarColors(val border: Color, val background: Color, val icon: Color)
```****